### DISCUSSION TITLE
qemu-system-aarch64: Property 't8030-machine.trustcache-filename' not found

### DISCUSSION THREAD

**Discussion by Anonymous941** [ChefKissInc/Inferno#28] [Q&A] [answered]
When trying to follow the instructions on [the old project's wiki](https://github.com/TrungNguyen1909/qemu-t8030/wiki/Bringing-up-the-emulator#auto-boot), which are the only instructions I could find:

```plain
qemu-system-aarch64: Property 't8030-machine.trustcache-filename' not found
```

Removing that property gives another error:

```plain
qemu-system-aarch64: Property 't8030-machine.ticket-filename' not found
```

Removing that also gives me another error:

```plain
qemu-system-aarch64: Could not load data from file 'kernelcache.research.iphone12b'
```

These are probably just name changes, but I have no idea what the new names are

**Comment by VisualEhrmanntraut** [comment, marked as answer]
just remove the `-filename` part from the property names

**Reply by Arbifoxx** [reply]
Besides the -filename, what else is different in the command?

**Reply by VisualEhrmanntraut** [reply]
Don't recall anything else other than that and two new properties (`seprom`, `sepfw`, only needed if you intend to emulate the SEP instead of simulate it, that part is not functional enough yet)

**Reply by Arbifoxx** [reply]
I was able to get the seprom, is the sepfw inside the ipsw?

**Reply by VisualEhrmanntraut** [reply]
It's one labelled sep-firmware and afaik it's encrypted with a key wrapped with SEP's GID key. As I mentioned there's no reason to use this feature right now as it does not work enough yet.

**Reply by Anonymous941** [reply]
Now I get the error:

```plain
qemu-system-aarch64: Could not load data from file 'kernelcache.research.iphone12b'
```

**Reply by VisualEhrmanntraut** [reply]
That means either the path doesn't exist or you pointed it to a bad file

**Reply by Anonymous941** [reply]
Sorry about that, I thought it was compiled into QEMU, not an actual file.  It turns out I was just in the wrong directory
